### ISSUE TITLE
Figure out how to discover Android core library page

### ISSUE THREAD

**Issue by langsmith** [mapbox/android-docs#803]
Love the new design of the docs site, @mapbox/docs .

Currently, it seems impossible to find a way to get to https://www.mapbox.com/android-docs/core/overview/ which was possible in the previous layout with the upper left-hand corner dropdown.  cc @electrostat @andrlee 

Other thing that's on my mind is how far down/buried the link to Mapbox Plugins for Android is on https://www.mapbox.com/documentation  ☹️ . Android devs wouldn't know to go to `All Docs` and then scroll all the way down.  I doubt I'll be able to convince you to put it in the `Maps` window 😄 
<img width="395" alt="screen shot 2019-01-08 at 4 49 33 pm" src="https://user-images.githubusercontent.com/4394910/50868702-9097c180-1365-11e9-8b2b-287bca49e4b6.png">

Guess I'll pepper links/mentions to it throughout https://www.mapbox.com/android-docs/maps/overview/. Perhaps below [the XML attributes section](https://www.mapbox.com/android-docs/maps/overview/#mapview-xml-attributes)

**Comment by colleenmcginnis** [comment]
Thanks @langsmith! I'm sure we'll hit a few bumps while we're rolling out some new infrastructure. Thanks for letting us know about this so quickly! 

>Currently, it seems impossible to find a way to get to https://www.mapbox.com/android-docs/core/overview/ which was possible in the previous layout with the upper left-hand corner dropdown.

We should definitely add this to www.mapbox.com/documentation.

>Other thing that's on my mind is how far down/buried the link to Mapbox Plugins for Android is on https://www.mapbox.com/documentation ☹️ . 

Something has to be at the bottom of the page 😬 and since I _think_ plugins cannot be used without the Maps SDK, it makes sense to me for plugins to sit further down the page. 

>Android devs wouldn't know to go to All Docs and then scroll all the way down. 

This is a fair point. Maybe with the introduction of subdomains, we could consider adding a docs.mapbox.com/android landing page just for orienting Android developers to tools available to them. 

>I doubt I'll be able to convince you to put it in the Maps window 😄

Actually, I would consider adding a `Plugins` item to the top-level nav since we have plugins for all platforms. @katydecorah what do you think? 

In summary I think we should:

1. Tomorrow: Add Core library to mapbox.com/documentation
2. This week: Discuss if we should add a `Plugins` item to the top-level nav in the docs page shell
3. In the next few weeks: Scope out platform-specific landing pages after subdomain work is finished

**Comment by langsmith** [comment]
>Something has to be at the bottom of the page 😬

`:android_is_not_impressed:`

>  I would consider adding a Plugins item to the top-level nav since we have plugins for all 

hm. I figured the top level for was for major product offering groups. `Maps`, `Search`, etc.

Another possibility could be to make `Plugins` have its own separate "horizontal section" on https://www.mapbox.com/documentation/ like `Maps`, `Navigation`, `Search`, and `Studio`.

**Comment by langsmith** [comment]
Keeping this open so that discussion above stays visible.

**Comment by langsmith** [comment]
Closing as this is resolved with https://github.com/mapbox/www.mapbox.com/pull/3830. If the remaining `Plugins` talk towards the bottom of https://github.com/mapbox/android-docs/issues/803#issuecomment-452520898 is important enough, please open a new ticket.
